src/WorkerFarm.js

```diff
@@ -2,8 +2,10 @@ const {EventEmitter} = require('events');
 const os = require('os');
 const Farm = require('worker-farm/lib/farm');
 const promisify = require('./utils/promisify');
+const path = require('path');
 
 let shared = null;
+const SYNC_TRESHOLD = 50000; // Treshold to start workers in bytes
```

Comment:
Curious how you came up with this threshold 😜 

Also, is it faster to compute file size or just count the number of files? Which is a better heuristic?

Reply:
Gah. Same comment, same time!

Reply:
I'm pretty sure it'll be faster to count them, i'll rework it, to also include node_modules and do some benchmarks comparing count vs size